flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/BoundedBlockingSubpartitionTest.java

```diff
@@ -99,8 +93,8 @@ ResultSubpartition createFailingWritesSubpartition() throws Exception {
 
 	// ------------------------------------------------------------------------
 
-	static Path tmpPath() throws IOException {
-		return new File(TMP_DIR.newFolder(), "subpartition").toPath();
+	static Path tmpPath() {
+		return fileChannelManager.createChannel().getPathFile().toPath();
```

Comment:
This commit is strictly speaking not necessary, correct? If so, I'd like to exclude it to prevent merge conflict with #8880.

Reply:
It seems a bit redundant for having both `fileChannelManager` and `TMP_DIR` meanwhile for the same function. I am not sure whether only this change would bring conflicts, because we also change the other parts of this class. If you confirm, I could revert this change.

Reply:
It is true that there are likely other merge conflicts too, but I disagree with your implied conclusion that this means we don't have to worry about them. More merge conflicts => more time spent on fixing them.

I'm not saying that we shouldn't make this change overall, but let's do it at a later date, when merging it doesn't interfere with other people's critical work.